src/StudyEntities/Candidate/Alias.php

```diff
@@ -0,0 +1,69 @@
+<?php declare(strict_types=1);
+/**
+ * Implementation of an Alias value object that is used as a shorthand to
+ * refer to a Project or Site in LORIS.
+ *
+ * PHP Version 7
+ *
+ * @category StudyEntities
+ * @package  LORIS
+ * @author   John Saigle <john.saigle@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+namespace LORIS\StudyEntities\Candidate;
+
+/**
+ * ValidatableIdentifier with the default constructor that call validate
+ *
+ * @category StudyEntities
+ * @package  LORIS
+ * @author   John Saigle <john.saigle@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+abstract class Alias extends ValidatableIdentifier
+{
+    /*
+     * The maxiumum length of the identifier.
+     *
+     * @var int
+     */
+    public const MIN_LENGTH = 3;
+
+    /*
+     * The minimum length of the identifier.
+     *
+     * @var int
+     */
+    public const MAX_LENGTH = 3;
```

Comment:
Why are all types of aliases always exactly 3 characters long?

Reply:
I wanted to use constants and just picked a number. Is there a better range?

Reply:
Why is there a range that needs to be enforced for aliases?

Reply:
I guess there doesn't need to be one enforced